### PR TITLE
docs-13092-acm-experience-experience-builder-page-not-default-jh

### PR DESCRIPTION
**Pull request by JennyHajee**
Updated the Diagnostics topic with steps to configure Login & Registration defaults  back to Experience Builder Page.

### CODE REVIEW

File: modules/ROOT/pages/diagnostics.adoc [hunk 1]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
+If a community was created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages will display and not the *Experience Builder Page*.
+
+After the items from *Community profile* through *Account created* are configured
+, fix the diagnostics for the *Account assigned*:
+
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name.
+. The *Login and Registration* page shows the community account in the *Accounts* field.
+. From the top, right of the community page, click the refresh icon. The *Experience Builder Page* is set for *Login Page Type*, *Forgot Password*, and *Registration Page Type* fields.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/diagnostics.adoc]:
```suggestion
. From the top, right of the community page, click the refresh icon. The *Experience Builder Page* is set for *Login Page Type* and *Registration Page Type* fields.
```

File: modules/ROOT/pages/diagnostics.adoc [hunk 2]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
-. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign* to verify that the account is assigned correctly.
-. Fix the remaining items in the list.
+If a community was created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages will display and not the *Experience Builder Page*.
+
+After the items from *Community profile* through *Account created* are configured
+, fix the diagnostics for the *Account assigned*:
+
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
+. From the search results, select the community name.
```
Comment (ferferrarazzo) [inline modules/ROOT/pages/diagnostics.adoc]:
```suggestion
. From the search results, select the community name 
```
+ account (for example: 'CommunityName Account'.)

**Review by ferferrarazzo — CHANGES_REQUESTED** [review]

**Review by ferferrarazzo — DISMISSED** [review]

File: modules/ROOT/pages/diagnostics.adoc [hunk 3]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
-. In the control panel, click *Community Administration* > *Login & Registration*.
-. Scroll to the section *Registration Page Configuration*.
-. Enable *Allow customers and partners to self-register*.
-. Set *Registration Page Type* to *Default Page*.
-. Select your community's member user as the value for *Profile*.
-. Enter your community's Salesforce account name as the value for *Account*.
-. Click *Save*.
-. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign* to verify that the account is assigned correctly.
-. Fix the remaining items in the list.
+If a community was created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages will display and not the *Experience Builder Page*.
```
Comment (kbostrom) [inline modules/ROOT/pages/diagnostics.adoc]:
```suggestion
If a community is created but not published, when diagnostics *Fix All* is run for the member user, the *Default Page* for the *Login and Registration* pages displays instead of the *Experience Builder Page*.
```

File: modules/ROOT/pages/diagnostics.adoc [hunk 4]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
+
+After the items from *Community profile* through *Account created* are configured
+, fix the diagnostics for the *Account assigned*:
```
Comment (kbostrom) [inline modules/ROOT/pages/diagnostics.adoc]:
```suggestion
After the items from *Community profile* through *Account created* are configured, fix the diagnostics for the *Account assigned*:
```

File: modules/ROOT/pages/diagnostics.adoc [hunk 5]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
+
+. In the diagnostics page, next to the *Member User* item *Account assigned*, click *Assign*.
+. From the *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
```
Comment (kbostrom) [inline modules/ROOT/pages/diagnostics.adoc]:
```suggestion
. From *Administration Community* > *Accounts*, click the search icon, enter the name of the community, and click *GO!*.
```

File: modules/ROOT/pages/diagnostics.adoc [hunk 6]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
+. From the search results, select the community name. The account name is the community name + account, for example, MyCommunityName Account.
```
Comment (kbostrom) [inline modules/ROOT/pages/diagnostics.adoc]: If the user is selecting the community name, do we really need to define the account name?
Maybe reword this if so, to something like
```suggestion
. From the search results, select the community name from the list of account names, which are displayed in the format MyCommunityName Account.
```

File: modules/ROOT/pages/diagnostics.adoc [hunk 7]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
+. The *Login and Registration* page shows the community account in the *Accounts* field.
```
Comment (kbostrom) [inline modules/ROOT/pages/diagnostics.adoc:67]: This is not a step, because there is no user action. It should be a plain paragraph at the same level as the previous step:

. From the search results, select the community name from the list of account names.
+
The *Login and Registration* page shows the community account in the *Accounts* field.

File: modules/ROOT/pages/diagnostics.adoc [hunk 8]
```diff
@@ -57,14 +57,14 @@ These configuration options work only for the default API Community Manager conf
...
+. From the top, right of the community page, click the refresh icon. The *Experience Builder Page* is set for *Login Page Type* and *Registration Page Type* fields.
```
Comment (kbostrom) [inline modules/ROOT/pages/diagnostics.adoc]:
```suggestion
. From the top right of the community page, click the refresh icon.
+
The *Experience Builder Page* is set for *Login Page Type* and *Registration Page Type* fields.
```

**Review by kbostrom — DISMISSED** [review]
PR; should have been ER

**Review by kbostrom — APPROVED** [review]
PR
